UserController - Initial endpoints implementation (JDP200101-5)

deseff opened the pull request.

deseff committed:
UserController - Initial endpoints implementation (JDP200101-5)

MARudnicki commented on src/main/java/com/kodilla/ecommercee/UserController.java:
```diff
@@ -1,4 +1,26 @@
+    @RequestMapping(method = RequestMethod.GET, value = "getToken")
+    public void getToken(Long userId) {
+
+    }
```
metoda getToken nie powinna byc void'em - powinna zwracac token

MARudnicki replied:
TokenDto powinno byc zwracanym typem

deseff committed:
UserController - Initial endpoints implementation (JDP200101-5)
- TokenGenerator added / in response to change request from review

MARudnicki commented on src/main/java/com/kodilla/ecommercee/UserController.java:
```diff
@@ -1,4 +1,29 @@
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("v1/user")
```
z tego co widze w pozostalych PR'ach, obowiazuje u Was konwencja z l. mnoga - 
users, products, groups

MARudnicki replied:
dodatkowo, zgodnie z uwagami w 
https://github.com/pplutap/JDP-2001-01/pull/4
https://github.com/pplutap/JDP-2001-01/pull/2

brakuje jeszcze elementu `project`

MARudnicki commented on src/main/java/com/kodilla/ecommercee/UserController.java:
```diff
@@ -1,4 +1,29 @@
 public class UserController {
+
+    @RequestMapping(method = RequestMethod.POST, value = "createUser")
+    public void createUser(String userDto) {
```
Tutaj powinno byc UserDto jako parametr, nie String

MARudnicki commented on src/main/java/com/kodilla/ecommercee/UserController.java:
```diff
@@ -1,4 +1,29 @@
+    }
+
+    @RequestMapping(method = RequestMethod.PUT, value = "blockUser")
+    public void blockUser(String userDto) {
```
Tutaj parametrem powinno byc Dto z id'kiem uzytkownika

deseff committed:
UserController - Initial endpoints implementation (JDP200101-5)
- TokenGenerator improved (shortened)
- UserDto created
- UserController changed - UserDto usage
- 'spring.jpa.hibernate.naming.strategy=org.hibernate.cfg.ImprovedNamingStrategy' removed from application.properties due to error caused
